apps/condo/domains/organization/schema/FindOrganizationsByTinService.test.js

```diff
@@ -0,0 +1,333 @@
+
+            const [o10nWithServiceUserOnly] = await registerNewOrganization(serviceClient, { tin, country: DEFAULT_ENGLISH_COUNTRY })
+
+            const [result] = await findOrganizationsByTinByTestClient(registeredStaffClient, { tin })
+            expect(result.organizations).toHaveLength(2)
+            expect(result.organizations).toEqual(expect.arrayContaining([
+                { id: o10nWithAdministrator.id, name: o10nWithAdministrator.name },
+                { id: o10nWithAdministrator2.id, name: o10nWithAdministrator2.name },
+            ]))
+        })
+
+        test('Requests should be logged in "FindOrganizationsByTinLog"', async () => {
+            const staffClient = await makeClientWithStaffUser()
+            const tin1 = String(generateTin())
+            const tin2 = String(generateTin())
+
+            const logs1 = await FindOrganizationsByTinLog.getAll(adminClient, { user: { id: staffClient.user.id } })
+            expect(logs1).toHaveLength(0)
+
+            const [result] = await findOrganizationsByTinByTestClient(staffClient, { tin: tin1 })
+            expect(result).toEqual({ organizations: [] })
+
+            const logs2 = await FindOrganizationsByTinLog.getAll(adminClient, { user: { id: staffClient.user.id } })
+            expect(logs2).toHaveLength(1)
+            expect(logs2).toEqual(expect.arrayContaining([
+                expect.objectContaining({
+                    tin: tin1,
+                    userPhone: staffClient.userAttrs.phone,
+                    userEmail: staffClient.userAttrs.email,
+                }),
+            ]))
+
+            const [result2] = await findOrganizationsByTinByTestClient(staffClient, { tin: tin2 })
+            expect(result2).toEqual({ organizations: [] })
+            const [result3] = await findOrganizationsByTinByTestClient(staffClient, { tin: tin1 })
+            expect(result3).toEqual({ organizations: [] })
+
+            const logs3 = await FindOrganizationsByTinLog.getAll(adminClient,
+                { user: { id: staffClient.user.id } },
+                { sortBy: ['createdAt_ASC'] }
+            )
+            expect(logs3).toHaveLength(3)
+            expect(logs3).toEqual([
+                expect.objectContaining({
+                    tin: tin1,
+                }),
+                expect.objectContaining({
+                    tin: tin2,
+                }),
+                expect.objectContaining({
+                    tin: tin1,
+                }),
+            ])
+        })
+
+        test('should throw error if find by unavailable tin', async () => {
+            const staffClient = await makeClientWithStaffUser()
+
+            const logs1 = await FindOrganizationsByTinLog.getAll(adminClient, { user: { id: staffClient.user.id } })
+            expect(logs1).toHaveLength(0)
+
+            await expectToThrowGQLError(async () => {
+                await findOrganizationsByTinByTestClient(staffClient, { tin: '0000000000' })
```

Comment:
If I don't have this tin in `conf.UNAVAILABLE_TINS_FOR_ORGANIZATIONS_SEARCH` then the test will fail.
Maybe use here `UNAVAILABLE_TINS` and get first element from it? And fill it like this: `['000...', ...conf.UNAVAILABLE_TINS_FOR_ORGANIZATIONS_SEARCH]`

Reply:
Added the necessary envs when running tests